src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -210,13 +211,17 @@
                      :collectible
                      :token-display-name
                      :amount
-                     (when (= transaction-type :collectible) :tx-type))})))
+                     (when (contains? #{:collectible :collectible-multi} transaction-type) :tx-type))})))
 
 (rf/reg-event-fx
  :wallet/set-collectible-to-send
  (fn [{db :db} [{:keys [collectible current-screen]}]]
    (let [collection-data    (:collection-data collectible)
          collectible-data   (:collectible-data collectible)
+         contract-type      (:contract-type collectible)
+         tx-type            (if (= contract-type constants/contract-type-erc-1155)
+                              :collectible-multi
```

Comment:
I know there will be cases where we sent multiple ERC721 tokens in the same multi-transaction, for example batch send for community tokens which are ERC-721 but treated as the same token. Is `:collectible-multi` some defined naming in the wallet domain? If not, I would suggest to just specify the token standard in the tx-type i.e. `:collectible-erc721` or `:collectible-erc1155`, IMO more understandable and extensible naming. WDYT?

Reply:
Yeah that makes sense.